API/Startup.cs

```diff
@@ -53,8 +53,10 @@ public void ConfigureServices(IServiceCollection services)
             services.AddControllers();
             services.Configure<ForwardedHeadersOptions>(options =>
             {
-                options.ForwardedHeaders =
-                    ForwardedHeaders.XForwardedFor | ForwardedHeaders.XForwardedProto;
+                options.ForwardedHeaders = ForwardedHeaders.All;
+                foreach(var proxy in _config.GetSection("KnownProxies").AsEnumerable().Where(c => c.Value != null)) {
```

Comment:
Will this code fail if KnowProfixes isn't set in the appsettings.json?

Reply:
It works when the section "KnownProxies" is missing. GetSection() returns an empty IConfigurationSection and enventually an empty enumerable, so no action is then taken.

Reply:
Also, I tested that case, just to be sure.